wagtail/admin/templates/wagtailadmin/generic/form.html

```diff
@@ -4,8 +4,9 @@
 
 {% block main_header %}
     {% if breadcrumbs_items %}
-        <div class="w-header nice-padding w-mt-8">
-            <h2 class="w-header__title" id="header-title">
+        <div class="nice-padding w-mt-8">
+            <h2 class="w-relative w-h1" id="header-title">
+                {% icon classname="w-absolute w-top-1 -w-left-11 w-max-w-[1em] w-max-h-[1em]" name=header_icon %}
```

Comment:
```suggestion
                {% icon classname="initial w-absolute w-top-1 -w-left-11" name=header_icon %}
```

I don’t think we need this to be a `max-width` specifically? I’d also feel more comfortable avoiding absolute positioning altogether for this. Possibly use CSS so we can have separate values for a negative left margin to offset the icon, and the gap.

Reply:
I think I tried using `w-w-4` but it didn't quite work for some reason. This is adapted from the existing styles for w-header just so that we're not using its styles directly, but yes I can look into simplifying this later. Thanks!